Classes/DDOSLogger.h

```diff
@@ -36,3 +36,9 @@ API_AVAILABLE(macos(10.12), ios(10.0), watchos(3.0), tvos(10.0))
 @property (class, readonly, strong) DDOSLogger *sharedInstance;
 
 @end
+
+@interface DDOSLogger (Variations)
+- (instancetype)withSubsystem:(NSString *)string;
```

Comment:
With the initializer, I'd remove these methods. They only cause confusion. Will they return the same instance? Will they create a new one (copy)?
I think most people will use this logger like the file logger: Create one (with a subsystem / category combination) and then add it.